src/main/java/nl/tudelft/lifetiles/Launcher.java

```diff
@@ -50,7 +51,7 @@
      * @param args
      *            The program arguments
      */
-    public static void main(final String[] args) {
+    public static void main(final String... args) {
```

Comment:
Yeah, I didn't saw that one before so I looked it up. It seems to be a working alternative.